app/javascript/dashboard/routes/login/Login.vue

```diff
@@ -149,7 +149,17 @@ export default {
           }
 
           if (response && response.status === 401) {
-            this.showAlert(this.$t('LOGIN.API.UNAUTH'));
+						const { errors } = response.data;
+            if (
+              errors &&
```

Comment:
Move this to a variable since more than 3 conditions are used

Reply:
Yeah, done.
So, can I know if you are planning the hotfix release in the new few hours? If 1.14.0 broke a feature like inbox_email it should be a priority for the hotfix to go out ASAP right?